core/blockchain.go

```diff
@@ -992,42 +961,46 @@ func (bc *BlockChain) insertBlock(block *types.Block, writes bool) error {
 	}
 
 	// Write the block to the chain and get the status.
-	// writeBlockWithState creates a reference that will be cleaned up in Accept/Reject
-	// so we need to ensure an error cannot occur later in verification, since that would
-	// cause the referenced root to never be dereferenced.
-	status, err := bc.writeBlockWithState(block, receipts, logs, statedb)
-	if err != nil {
+	// writeBlockWithState (called within writeBlockAndSethead) creates a reference that
+	//  will be cleaned up in Accept/Reject so we need to ensure an error cannot occur
```

Comment:
Can we remove the extra space before `will` here? I noticed while reviewing and just added a PR to coreth to remove it as well. https://github.com/ava-labs/coreth-internal/pull/596